runtime/core/exec_aten/util/tensor_util.h

```diff
@@ -484,6 +486,30 @@ inline bool tensor_is_type(
   return true;
 }
 
+inline bool tensor_is_type(
+    executorch::aten::Tensor t,
+    const std::vector<executorch::aten::ScalarType>& dtypes) {
+  if (std::find(dtypes.begin(), dtypes.end(), t.scalar_type()) !=
+      dtypes.end()) {
+    return true;
+  }
+
+  std::stringstream dtype_ss;
+  for (size_t i = 0; i < dtypes.size(); i++) {
+    if (i != 0) {
+      dtype_ss << ", ";
+    }
+    dtype_ss << torch::executor::toString(dtypes[i]);
+  }
+
+  ET_LOG_MSG_AND_RETURN_IF_FALSE(
+      false,
+      "Expected to find one of %s types, but tensor has type %s",
+      dtype_ss.str().c_str(),
+      torch::executor::toString(t.scalar_type()));
+  return false;
```

Comment:
No need to use "return if false" when it's always false; a basic log macro and a return statement is more clear.
```suggestion
  ET_LOG(Info
      "Expected to find one of %s types, but tensor has type %s",
      dtype_ss.str().c_str(),
      torch::executor::toString(t.scalar_type()));
  return false;
```